clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/servergroup/ResizeDcosServerGroupAtomicOperation.groovy

```diff
@@ -46,10 +46,7 @@ class ResizeDcosServerGroupAtomicOperation implements AtomicOperation<Void> {
 
     def app = new App()
 
-    app.with {
-      id = appId.toString()
-      instances = size
-    }
+    app.instances = description.targetSize
```

Comment:
I actually didn't think about this when I made the change but it doesn't look like it stop this from working, just tested it locally.